fluid/PaddleCV/image_classification/README_ngraph.md

```diff
@@ -0,0 +1,40 @@
+
+# PaddlePaddle inference and training script
+This directory contains configuration and instructions to run the PaddlePaddle + nGraph for a local training and inference.
+
+# How to build PaddlePaddle framework with NGraph engine
+In order to build the PaddlePaddle + nGraph engine and run proper scripti,  follow up a few steps:
+1. Install PaddlePaddle project
+2. set env exports for nGraph and OMP
+3. run the inference/training script
+
+Curently supported models:
+* ResNet50 (inference and training).
+
+Short description of aforementioned steps:
+
+## 1. Install PaddlePaddle
+Follow PaddlePaddle [installation instruction](https://github.com/PaddlePaddle/models/tree/develop/fluid/PaddleCV/image_classification#installation) to install PaddlePaddle. If you build PaddlePaddle yourself, please use the following cmake arguments and ensure to set -DWITH_NGRAPH=ON.  
```

Comment:
This link finally jumps to v1.3, it is suggested to provide flexible link for user, such as https://github.com/PaddlePaddle/FluidDoc/tree/develop/doc/fluid/beginners_guide/install